src/peps_util.jl

```diff
@@ -202,7 +210,7 @@ function buildN(A::PEPS, L::Environments, R::Environments, IEnvs, row::Int, col:
         ci = commonindex(A[row, col], A[row, col-1])
         workingN *= multiply_side_ident(A[row, col], ci, copy(L.I[row])) 
     end
-    workingN *= ϕ
+    workingN *= phi
```

Comment:
What's wrong with ϕ?

Reply:
It's not a huge deal to use \phi I don't think, but I was thinking that for public-facing, open-source codes, we want to be sure of at least two things:
- the code can be viewed and properly formatted on a wide range of devices and in a wide range of text editors, such as on older cluster machines, various peoples' laptops etc.
- people contributing to the code know how to type all of the characters used; I could imagine people less plugged into the Julia ecosystem not knowing how to type special characters and having to spend time looking this up

Reply:
If we can be really sure of those, then I think it's ok to use special characters to a limited extent. I've actually been sneakily removing \in and other special characters from ITensors.jl, though, because I'm not convinced it's a choice that will age well.

Reply:
This is entirely in internal code, though, and are there really so many people trying to read this code outside of a browser or a modern code editor like Atom/VSCode? Even in vim this renders natively.

Reply:
I changed it back just to save us both time from discussing it too much more. It's not something I feel too strongly about. Ok hope the rest of the changes look ok. I have a lot more, this was just the first PR.